ext/http/00_serve.ts

```diff
@@ -443,15 +470,16 @@ function fastSyncResponseOrStream(req, respBody, status, innerRequest) {
  * This function returns a promise that will only reject in the case of abnormal exit.
  */
 function mapToCallback(context, callback, onError) {
-  const signal = context.abortController.signal;
-
   return async function (req) {
+    const abortController = new AbortController();
```

Comment:
Are we sure we want to create one unconditionally?

Reply:
I don't have a great way to do this conditionally right now -- we'd have to refactor `Request` a bit for that. For now, impact seems small (not zero).

Reply:
> Are we sure we want to create one unconditionally?

Yes)